src/components/resource-picker/storage-location.yml

```diff
@@ -1,18 +1,18 @@
+filterPlaceholder:
+  en: Filter
+  de: Filter
 filesView_list:
-  en: List
-  de: Liste
+  en: List view
+  de: Listenansicht
 filesView_grid:
-  en: Grid
-  de: Grid
+  en: Grid view
+  de: Grid-Ansicht
```

Comment:
We could go with "Kachelansicht" in German here

Reply:
Went with, thanks!